### DISCUSSION TITLE
ifit with echelon bike what what do I need?

### DISCUSSION THREAD

**Discussion by davcho80** [cagnulein/qdomyos-zwift#2189] [Q&A]
I'm trying to see what I missed. I tried several methods to get my Echelon bike recognized in IFIT. Qzapp on ios and ifit on android, qzapp on android and ifit on ios but nothing seems to work. It never appears in the ifit application even if it is plugged into qzapp

thank you

**Comment by cagnulein** [comment]
@davcho80 did you enable the virtual ifit on experimental settings? use qz on iphone and ifit on android
